examples/react-cra/src/components/Search.tsx

```diff
@@ -0,0 +1,40 @@
+import type { NodeSearchResult } from '@stoplight/elements-dev-portal';
+import { Search as ElementsSearch, useGetNodes } from '@stoplight/elements-dev-portal';
+import * as React from 'react';
+
+export type SearchType = {
+  projectIds: string[];
+  workspaceId: string;
+};
+export const Search = ({ projectIds, workspaceId }: SearchType) => {
+  const [search, setSearch] = React.useState('');
+  const [open, setOpen] = React.useState(false);
+  const { data } = useGetNodes({
+    search,
+    projectIds,
+    workspaceId,
+  });
+
+  const handleClose = () => {
+    setOpen(false);
+    setSearch('');
+  };
+
+  const handleClick = (data: NodeSearchResult) => {
+    console.log('clicked', data);
```

Comment:
It wouldn't be that much more work to actually make it send you to a proper article url.

Reply:
yes, but even then I'm not sure exactly how to make it work. It looks like internally the API component uses `react-router`... I'm not sure how to hook the API component and this callback together.

Reply:
I need some help with this conceptually. I was originally thinking to just use `window.location.href` and set it to the node, but will need to then get the workspace slug from the encoded ID that comes back in `data.` I have been searching to see if I could find where something like that is already happening anywhere but not sure it is. This seems like not what Will was thinking...

Reply:
I have a naive approach I committed in 4771e98. I have a feeling this is the least efficient way to do it, but I am out of ideas. Would love to get some guidance if possible.